src/ui/EditTask.svelte

```diff
@@ -631,7 +631,11 @@ Availability of access keys:
             <!-- --------------------------------------------------------------------------- -->
             <!--  Done Date  -->
             <!-- --------------------------------------------------------------------------- -->
-            <label for="done">Done (<span class="accesskey">x</span>)</label>
+            {#if withAccessKeys}
+                <label for="done">Done (<span class="accesskey">x</span>)</label>
+            {:else}
+                <label for="done">Done</label>
+            {/if}
```

Comment:
For future reference....

I didn't think of this when reviewing the diff, but if we have any more like this - accesskeys that are not inside the label - it might be nice to have a helper function that takes the label and the access key, and decides whether or not to append the ` (x)` or whatever....

Reply:
Indeed, I thought about that too. And maybe even apply this to each label+input. This would also simplify some parts of the CSS.

I’ve not included that in this PR to keep it small)